thcrap/src/stack.cpp

```diff
@@ -390,17 +390,16 @@ int stack_remove_if_unneeded(const char *patch_id)
 			game_found |= patch_file_exists(&patch, js.c_str());
 		}
 	}
-	if (!game_found) {
-		stack.erase(patch_it);
```

Comment:
We want things like base_tsa to be loaded in something like vpatch so that versions.js can still be loaded, but we still want to cut off initialization of the the plugins